_includes/tutorials/connect-sink-timestamp/ksql/markup/dev/create-sink-connector.adoc

```diff
@@ -0,0 +1,7 @@
+Now create a JDBC sink connector with the code below. Notice no transforms will be necessary to the `eventTime` column because the column is already a supported `TIMESTAMP` type.
```

Comment:
WDYT about using the word `deploy` instead of `create`?  Goal being to differentiate jar installation as done earlier with `confluent-hub install`.

Reply:
changed